### ISSUE TITLE
We should move Authentication on/off toggles out of their config forms

### ISSUE THREAD

**Issue by mazameli** [metabase/metabase#5555]
This doesn't need to happen for 0.25, but one way we could improve the UX for the Google Sign-in and LDAP auth settings is:

1. move the enabled/disabled toggle for LDAP from the configuration form to the summary card:

I.e., from here:
![screen shot 2017-07-24 at 10 41 58 am](https://user-images.githubusercontent.com/2223916/28536637-1c98a594-705d-11e7-8965-6c049a9dbad4.png)

to here:
![screen shot 2017-07-24 at 10 42 09 am](https://user-images.githubusercontent.com/2223916/28536631-169b30da-705d-11e7-85d0-4b85293619e5.png)

Reason being, right now it's a bit confusing that you can fill in all the settings and hit save, and if you forgot to hit the toggle you might think LDAP is enabled. You might also have filled in the settings, then scroll back to the top and flip the toggle, not realizing you needed to hit Save Changes, then navigate away and unknowingly lose your changes. Separating the toggle out lets the form just be "here are my settings," and then the toggle becomes about whether or not those settings should take effect.

2. The Google Sign-in form currently doesn't have an on/off switch, so we could add one and put it on the Google Sign-in summary card to make the two forms behave the same.

**Comment by lindsay-stevens** [comment]
@mazameli is this ticket still valid? I think the proposal is how the UI looks now (v0.34+).

**Comment by mazameli** [comment]
Yeah, it's still valid. The screenshots show how the current UI looks, but the proposal I wrote out in the description is to move the on/off toggles on to the cards depicted in the second screenshot, e.g. somewhere next to the "Configure" button.
